release/background.js

```diff
@@ -212,7 +212,7 @@ function updateContextMenus()
     //     WE NEED TO CALL THIS UPON PREFS CHANGES FOR THE TIME BEING TO ENSURE IT'S CREATED BEFORE IT'S NEEDED.
     //     Rejig whenever the above bug is included (allows dynamic menu updating) - not in FF dev as of end Oct '17!!
 
-    /// let isUrl = (activeSelection.search(/^(http|ftp)/) >= 0);
```

Comment:
Ew, I need to figure out if that should include **file:** and **javascript:** before they get their act together and fix dynamic menus ...

Reply:
Were you able to progress on this matter?

Reply:
Soz, not yet. Very little spare time ATM, used what I had recently to sort out Thunderbird version compatibility.